src/mono/mono/mini/mini-llvm.c

```diff
@@ -7477,7 +7526,12 @@ process_bb (EmitContext *ctx, MonoBasicBlock *bb)
 			mono_llvm_build_aligned_store (builder, first_elem, dst, FALSE, 1);
 			break;
 		}
-
+		case OP_SSE_MOVNTPS: {
+			printf ("X: %s\n", mono_method_get_full_name (cfg->method));
+			LLVMValueRef store = mono_llvm_build_aligned_store (builder, rhs, lhs, FALSE, 1);
```

Comment:
Shouldn't it be aligned to 16 since it has "Aligned" in its name **StoreAlignedNonTemporal**? (https://godbolt.org/z/wzW4ZA)

Reply:
1 is what llvm tests use.